controllers/admin/AdminCartRulesController.php

```diff
@@ -538,6 +538,7 @@ public function ajaxProcess()
 				OR `firstname` LIKE "%'.pSQL($search_query).'%"
 				OR `lastname` LIKE "%'.pSQL($search_query).'%"
 			)
+			'.Shop::addSqlRestriction(true).'
```

Comment:
hi @christianverardi 
this should be Shop::addSqlRestriction(Shop::SHARE_CUSTOMER)
Could you please fix this argument ?